services/apps/src/apps/persistence/app_metadata.clj

```diff
@@ -710,3 +719,28 @@
                            {:u.username username})
                        {:aca.app_id (uuidify app-id)
                         :l.id       [not= (user-favorite-subselect :w.root_category_id faves-idx)]})))))
+
+(defn- list-duplicate-apps*
+  [app-name app-id category-id-set]
+  (select [:apps :a]
+          (fields :a.id :a.name :a.description)
+          (join [:app_category_app :aca] {:a.id :aca.app_id})
+          (where {(raw "trim(both from a.name)") (string/trim app-name)
```

Comment:
... an extra bit of possible improvement here could also be to check the equivalence more ways -- at least `lower(trim(name))` could be nice, and (though I don't know if this is possible in our DB) collapsing whitespace to one space, so something like `lower(regexp_replace(trim(name), '\s+', ' '))` or so.

Sadly we can't add an index to enforce this since it's multi-table and triggers are a pain.